src/sqlite-vss.cpp

```diff
@@ -479,10 +474,10 @@ static int write_index_insert(faiss::Index *index,
         sqlite3_stmt *stmt;
         char *sql = sqlite3_mprintf(
             "insert into \"%w\".\"%w_index\"(rowid, idx) values (?, ?)",
-            schema,
-            name);
+            index_vtab->schema,
+            index->name);
```

Comment:
should this should be `index_vtab->name`?

currently this patch breaks creating a`faiss_shadow` index.

```
SQLite Error [1]: no such table: main.`[?]_index in "insert into "main"."`[?]_index"(rowid, idx) values (?, ?)"
```

using `index_vtab->name` fixes with this change.